hw/ip/otbn/README.md

```diff
@@ -0,0 +1,41 @@
+# OpenTitan Big Number Accelerator (OTBN)
+
+This directory contains the implementation of the OpenTitan Big Number
+Accelerator (OTBN). OTBN is a coprocessor for asymmetric cryptographic
+operations like RSA or Elliptic Curve Cryptography (ECC).
+
+See https://docs.opentitan.org/hw/ip/otbn/doc/index.html for documentation on
+the current version of OTBN; documentation matching the code in this directory
+can be found in the `doc` directory.
+
+OTBN is currently in early development. Please ask questions and report issues
+through the [GitHub issue tracker](https://github.com/lowRISC/opentitan/issues).
+
+## Develop OTBN
+
+## Build the RTL implementation
+
+To build the RTL implementation of OTBN, do *not* pass the `OTBN_MODEL` flag to
+`fusesoc` and compile the simulation as usual. For example, the Verilator
+simulation can be compiled as follows.
+
+```sh
+fusesoc --cores-root=. run --target=sim --setup --build lowrisc:systems:top_earlgrey_verilator
+```
+
+## Run the OTBN hello world application
+
+```sh
+# Build the software
+ninja -C build-out/
+
+# Run the Verilator-based simulation
+./build/lowrisc_systems_top_earlgrey_verilator_0.1/sim-verilator/Vtop_earlgrey_verilator \
+  --meminit=rom,build-bin/sw/device/boot_rom/boot_rom_sim_verilator.elf \
+  --meminit=flash,build-bin/sw/device/examples/hello_otbn/hello_otbn_sim_verilator.elf
+```
+
+The `hello_otbn` application outputs diagnostic information to the UART, read it
+in simulation with e.g. `cat /dev/pts/10`. (Use the UART device shown when
+running the simulation.)
```

Comment:
This should be a `;` or `.` after UART. Also, I'd probably use "writes" rather than "outputs", which isn't really a verb in my book!